clang/examples/DPCT/Runtime/cudaMemcpy2DArrayToArray.cu

```diff
@@ -1,9 +1,11 @@
-void test(cudaArray_t a, size_t s1, size_t s2, cudaArray_const_t ac, size_t s3,
-          size_t s4, size_t s5, size_t s6, cudaMemcpyKind m) {
+void test(size_t s1, size_t s2, size_t s3, size_t s4, size_t s5, size_t s6) {
+  // TODO: src's type need to be changed to cudaArray_const_t
```

Comment:
fix the todo

Reply:
Now the cudaArray_const_t cannot be migrated correctly.